### ISSUE TITLE
Add a check of the bundle size to ensure it doesn't get too large

### ISSUE THREAD

**Issue by tilanka** [DeloitteDigitalAPAC/rhythm-ui#48]
We should ensure this library never gets too large by implementing a check against the bundle size. Can potentially do it via inspecting the outputs of `webpack-bundle-analyzer` but  otherwise there  are other npm packages that do this

Inspired by https://material-ui.com/guides/minimizing-bundle-size/

**Comment by ricominten** [comment]
@tilanka Just out of curiosity, what is considered `too large`?

**Comment by tilanka** [comment]
@ricominten good question, I think that's up to us to define really. A single component would be a few kB or less ideally

**Comment by jennasalau** [comment]
This is a good call out.

I'm not sure how we can measure this at a component level in this repo. Each component is its own package which means consumers can opt-in to only the components they need. It also means we are not using web-pack any where as this would live at the project level.

Perhaps we can write a script that measures the "file size" of each of the components "lib" folder?

**Comment by tilanka** [comment]
yeah that would be how it would have to work. I'll take a look at some options and come back with an approach

**Comment by jennasalau** [comment]
Atlassian has some documentation that might help accelerate your research.

https://atlaskit.atlassian.com/docs/guides/bundle-size

**Comment by christopher-hopper** [comment]
Thank you for your contributions. We are archiving this repository.
